TestVerifyOdsProjectProvisionThruProvisionApi fails on ODS box

**Describe the bug**
test fails with error message:
"status":500,"error":"Internal Server Error","message":"No quickstarters to delete are passed"

**To Reproduce**
Steps to reproduce the behavior:
1. Build new ODS box
2. run test TestVerifyOdsProjectProvisionThruProvisionApi
3. See error

**Expected behavior**
test should pass

**Affected version (please complete the following information):**
 - OpenShift: 3.11
 - OpenDevStack latest master

**Log Output (ensure to remove any confidential information like tokens, project names, etc.**
```
    amazon-ebs: --- FAIL: TestVerifyOdsProjectProvisionThruProvisionApi (337.57s)
    amazon-ebs:     verify-ods-project-provision-api_test.go:103: Failed to delete component: Execution of `provisioning-app-api.sh/delete component` for 'DELETE_COMPONENT/ODSVERIFY' failed:
    amazon-ebs:         StdOut: No config file found, assuming defaults, current dir: /home/openshift/opendevstack/ods-core/tests/ods-verify
    amazon-ebs:
    amazon-ebs:         Started provision project/component script with command (DELETE_COMPONENT)!
    amazon-ebs:
    amazon-ebs:         ... encoding basic auth credentials in base64 format
    amazon-ebs:
    amazon-ebs:         ... sending request to 'https://prov-app-ods.ocp.odsbox.lan' (output will be saved in file './response.txt' and headers in file './headers.txt')
    amazon-ebs:
    amazon-ebs:
    amazon-ebs:         create or update project, or delete component - DELETE_COMPONENT
    amazon-ebs:         ... DELETE_COMPONENT project request payload loaded from 'fixtures/create-project-request.json'
    amazon-ebs:
    amazon-ebs:         ... displaying payload file content:
    amazon-ebs:         {
    amazon-ebs:             "projectName": "ODSVERIFY",
    amazon-ebs:             "projectKey": "ODSVERIFY",
    amazon-ebs:             "description": "Prov App test",
    amazon-ebs:             "projectType": "default",
    amazon-ebs:             "bugtrackerSpace": true,
    amazon-ebs:             "platformRuntime": true
    amazon-ebs:         }
    amazon-ebs:         curl DELETE request successful...
    amazon-ebs:
    amazon-ebs:         ... displaying HTTP response body (content from './response.txt'):
    amazon-ebs:         {"timestamp":"2020-08-26T17:07:07.295+0000","status":500,"error":"Internal Server Error","message":"No quickstarters to delete are passed","path":"/api/v2/project"}
    amazon-ebs:         ... displaying HTTP response code
    amazon-ebs:         http_resp_code=500
    amazon-ebs:
    amazon-ebs:         something went wrong... endpoint responded with error code [HTTP CODE=500] (expected was 200)
    amazon-ebs:
    amazon-ebs:         StdErr:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
    amazon-ebs:                                          Dload  Upload   Total   Spent    Left  Speed
    amazon-ebs: 100   346    0   164  100   182    763    847 --:--:-- --:--:-- --:--:--   850
    amazon-ebs:
    amazon-ebs:         Err: exit status 1
    amazon-ebs: FAIL
    amazon-ebs: FAIL    github.com/opendevstack/ods-core/tests/ods-verify   350.181s
    amazon-ebs: FAIL
```

--- a/tests/utils/provisioning.go
+++ b/tests/utils/provisioning.go
@@ -72,7 +72,7 @@ func (api *ProvisionAPI) DeleteComponent() error {
 	stages, stderr, err := RunScriptFromBaseDir(
 		"tests/scripts/provisioning-app-api.sh",
 		[]string{"DELETE_COMPONENT"},
-		[]string{},
+		[]string{"PROVISION_FILE=fixtures/create-component-request.json"},
 	)
 
 	if err != nil {

